packages/schematics/angular/utility/index.ts

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
I'm wondering if there is a better name we can give this subpath. Most of the stuff here is related to workspaces, so should we consider exporting at `@schematics/angular/workspace`? What do you foresee adding in the future? Should we consider other subpaths for other common use cases?

Reply:
I think too many subpaths can lead to a lack of discoverability.  It can also lead to a large amount of import statements in schematics for only a small amount of imported values per module.  Many of these rules are frequently used together.  The dependency related rules and the eventual AST helpers will also be available here as well.

Reply:
Sounds reasonable, I mainly just wanted to make sure we were considering alternatives as well.